datafusion/tests/test_dataframe.py

```diff
@@ -164,16 +164,17 @@ def test_join():
         [pa.array([1, 2, 3]), pa.array([4, 5, 6])],
         names=["a", "b"],
     )
-    df = ctx.create_dataframe([[batch]])
+    df = ctx.create_dataframe([[batch]], "l")
```

Comment:
I had to alias these tables to fix an ambiguous column reference later in the test ... seems like an improvement